### ISSUE TITLE
Embedded Album cover art not loaded

### ISSUE THREAD

**Issue by rhauff1** [ampache/ampache#3537]
### Describe the bug

When adding new music to the database, the cover art is not automatically loaded, and now cannot load manually either.

### To reproduce
Add new music (with cover art embedded) to database
View album, no artwork is shown.
Click on tool on image to Edit/Find Art.
Under Cover Art Search, click "Get Art"
Artwork loads, and first image on left which should be embedded art (I believe) shows "Invalid"

### Screenshots

![Edit-Find_art](https://github.com/ampache/ampache/assets/75855702/06bd4c7e-8ff0-4bee-aaea-ccca0d88f572)
![Invalid_art](https://github.com/ampache/ampache/assets/75855702/24501e9b-4a62-4192-990c-680f7dfff25a)

## Environment

* Ampache version:  Ampache 5.6.0-release
* Web server + version:  Apache/2.4.52 (Ubuntu)
* Server operating system:  Ubuntu 22.04.2 LTS
* Client operating system:  Ubuntu 22.04.2 LTS

### Client type

* Web
* Ampache
* SubSonic

## Logs
Snippet of Ampache log that is generated by the above steps:

```2023-06-12T14:11:34-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:34-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:34-05:00 [roland] (Ampache\Module\System\Session) -> qh5vl3bonm2uht8t3e58chim6d has been extended to Mon, 12 Jun 2023 15:11:34 -0500 extension length 3600
2023-06-12T14:11:34-05:00 [roland] (ajax.server) -> Called for page: {index}
2023-06-12T14:11:34-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600694
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\System\Session) -> qh5vl3bonm2uht8t3e58chim6d has been extended to Mon, 12 Jun 2023 15:11:54 -0500 extension length 3600
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Album\ShowAction" for action "show"
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\System\Session) -> Session created: 60a34e9071b9875ed1e9ab7e741787ce
2023-06-12T14:11:54-05:00 [roland] (Ampache\Repository\Model\Browse) -> Show objects called for type {song}
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600714
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Image\ShowAction" for action "show"
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:54-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\System\Session) -> qh5vl3bonm2uht8t3e58chim6d has been extended to Mon, 12 Jun 2023 15:11:54 -0500 extension length 3600
2023-06-12T14:11:54-05:00 [roland] (ajax.server) -> Called for page: {browse}
2023-06-12T14:11:54-05:00 [roland] (browse.ajax) -> Called for action: {get_filters}
2023-06-12T14:11:54-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600714
2023-06-12T14:11:57-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:57-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:57-05:00 [roland] (Ampache\Module\System\Session) -> qh5vl3bonm2uht8t3e58chim6d has been extended to Mon, 12 Jun 2023 15:11:57 -0500 extension length 3600
2023-06-12T14:11:57-05:00 [roland] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Art\ShowArtDlgAction" for action "show_art_dlg"
2023-06-12T14:11:57-05:00 [ampache] (log.lib) -> [Error] Constant FILTER_SANITIZE_STRING is deprecated in file /opt/ampache/src/Module/Application/Art/ShowArtDlgAction.php(49)
2023-06-12T14:11:57-05:00 [ampache] (log.lib) -> [Error] Constant FILTER_SANITIZE_STRING is deprecated in file /opt/ampache/src/Module/Application/Art/AbstractArtAction.php(38)
2023-06-12T14:11:57-05:00 [roland] (Ampache\Module\System\Session) -> Session created: 2185d2a82fc4e52dc5e7844a10e26238
2023-06-12T14:11:57-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600717
2023-06-12T14:11:58-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:58-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\System\Session) -> qh5vl3bonm2uht8t3e58chim6d has been extended to Mon, 12 Jun 2023 15:11:58 -0500 extension length 3600
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Art\FindArtAction" for action "find_art"
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Constant FILTER_SANITIZE_STRING is deprecated in file /opt/ampache/src/Module/Application/Art/AbstractArtAction.php(38)
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\System\Session) -> Session created: 19b890805da246ef29435a49eccb3481
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Searching using:["db","tags","folder","lastfm","google"]
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Method used: db
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Method used: tags
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Method used: folder
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\FolderCollectorModule) -> gather_folder: Opening /audiovideo/music/Phantoms/Disconnect and checking for album Art
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Method used: lastfm
2023-06-12T14:11:58-05:00 [roland] (Ampache\Module\LastFm\LastFmQuery) -> search url : http://ws.audioscrobbler.com/2.0/?method=album.getInfo&artist=Phantoms&album=Disconnect+%5BExplicit%5D&api_key=fdf3edd2a8a6a642cf9302f8c599074a
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-06-12T14:11:58-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-06-12T14:11:59-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(40)
2023-06-12T14:11:59-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(51)
2023-06-12T14:11:59-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(68)
2023-06-12T14:11:59-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(82)
2023-06-12T14:11:59-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(91)
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/34s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/64s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/174s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/300x300/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/300x300/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\LastFmCollectorModule) -> LastFM: found image https://lastfm.freetls.fastly.net/i/u/300x300/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\ArtCollector) -> Method used: google
2023-06-12T14:11:59-05:00 [roland] (Ampache\Module\Art\Collector\GoogleCollectorModule) -> Search url: http://www.google.com/search?source=hp&tbm=isch&q=Phantoms%20Disconnect%20%5BExplicit%5D&oq=&um=1&ie=UTF-8&sa=N&tab=wi&start=0&tbo=1&imgsz=m
2023-06-12T14:12:00-05:00 [roland] (Ampache\Module\System\Core) -> Cannot create image from empty data
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/300x300/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/174s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/64s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/34s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600720
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Image\ShowAction" for action "show"
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Image\ShowAction" for action "show"
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Image\ShowAction" for action "show"
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/300x300/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/174s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/64s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(40)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(51)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(68)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(82)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(91)
2023-06-12T14:12:00-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600720
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
2023-06-12T14:12:00-05:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\Image\ShowAction" for action "show"
2023-06-12T14:12:00-05:00 [roland] (Ampache\Repository\Model\Art) -> CHECKING URL https://lastfm.freetls.fastly.net/i/u/34s/d5056db4a4aca93fb61c34885f66e7bc.jpg
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(63)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(73)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(89)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(102)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Cookie/Jar.php(111)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(40)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(51)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(68)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(82)
2023-06-12T14:12:00-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(91)
2023-06-12T14:12:00-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600720
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(40)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(51)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(68)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(82)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(91)
2023-06-12T14:12:01-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600721
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(40)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(51)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(68)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(82)
2023-06-12T14:12:01-05:00 [ampache] (log.lib) -> [Error] Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in file /opt/ampache/vendor/rmccue/requests/library/Requests/Utility/CaseInsensitiveDictionary.php(91)
2023-06-12T14:12:01-05:00 [roland] (Ampache\Module\System\Session) -> Writing to qh5vl3bonm2uht8t3e58chim6d with expiration 1686600721```

**Comment by rhauff1** [comment]
Here are my Gather Order settings:
; Art Gather Order
; Simply arrange the following in the order you would like
; Ampache to search. If you want to disable one of the search
; methods simply leave it out. DB should be left as the first
; method unless you want it to overwrite what's already in the
; database
; POSSIBLE VALUES (builtins): db tags folder lastfm musicbrainz google
; POSSIBLE VALUES (plugins): Amazon,TheAudioDb,Tmdb,Omdb,Flickr
; DEFAULT: db,tags,folder,spotify,musicbrainz,lastfm,google
art_order = "db,tags,folder,lastfm,google"

**Comment by barcoboy** [comment]
Trying to import cover art from tags not working here either.  Other import forms seem to be working.

**Comment by mitchray** [comment]
Seems to be fixed by #3572
